AK/Bitmap.h

```diff
@@ -22,12 +24,20 @@ class Bitmap : public BitmapView {
 public:
     Bitmap() = default;
 
-    Bitmap(size_t size, bool default_value)
-        : BitmapView(static_cast<u8*>(kmalloc(ceil_div(size, static_cast<size_t>(8)))), size)
-        , m_is_owning(true)
+    static ErrorOr<Bitmap> try_create(size_t size, bool default_value)
     {
         VERIFY(size != 0);
-        fill(default_value);
+        u8* data = static_cast<u8*>(kmalloc(ceil_div(size, static_cast<size_t>(8))));
+        if (!data)
+            return Error::from_errno(ENOMEM);
+        Bitmap bitmap { data, size, true };
+        bitmap.fill(default_value);
+        return bitmap;
+    }
+
+    static Bitmap must_create_but_fixme_should_propagate_errors(size_t size, bool default_value)
+    {
+        return MUST(try_create(size, default_value));
```

Comment:
Why not use `ErrorOr<>::release_value_but_fixme_should_propagate_errors` in the users?

Reply:
I don't see how that would make the code clearer. I'm not sure when to use which one, but since in this case, writing a short wrapper function will make the code shorter at the call-sites, I went with that.